deployment/sriov-network-operator/values.yaml

```diff
@@ -27,12 +27,14 @@ operator:
   resourcePrefix: "openshift.io"
   cniBinPath: "/opt/cni/bin"
   clusterType: "kubernetes"
+  metricsExporterPort: "9110"
```

Comment:
in some other places I can see you used 9808

Reply:
you're right. Fixed